net-dns/dnsdist/dnsdist-1.1.0.ebuild

```diff
@@ -0,0 +1,103 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+EGIT_REPO_URI="https://github.com/PowerDNS/pdns.git"
+
+if [[ ${PV} = 9999 ]]; then
+	ADDITIONAL_ECLASSES="autotools git-r3"
+fi
+
+inherit eutils flag-o-matic user ${ADDITIONAL_ECLASSES}
+
+DESCRIPTION="A highly DNS-, DoS- and abuse-aware loadbalancer"
+HOMEPAGE="http://dnsdist.org"
+
+if [[ ${PV} == 9999 ]]; then
+	SRC_URI=""
+	S="${WORKDIR}/${P}/pdns/dnsdistdist"
+else
+	SRC_URI="https://downloads.powerdns.com/releases/${P}.tar.bz2"
+	KEYWORDS="~amd64"
+fi
+
+LICENSE="GPL-2"
+SLOT="0"
+IUSE="dnscrypt luajit readline regex remote-logging +ssl test"
+RESTRICT="readline? ( bindist )"
+REQUIRED_USE="dnscrypt? ( ssl )"
+
+DEPEND="
+	>=dev-libs/boost-1.35:=
+	luajit? ( dev-lang/luajit:= )
+	!luajit? ( >=dev-lang/lua-5.1:= )
+	remote-logging? ( dev-libs/protobuf:= )
+	readline? ( sys-libs/readline:0= )
+	!readline? ( dev-libs/libedit:= )
+	regex? ( dev-libs/re2:= )
+	ssl? ( dev-libs/libsodium:= )
+"
+
+RDEPEND="${DEPEND}"
+
+[[ ${PV} == 9999 ]] && DEPEND+="
+	app-text/pandoc
+	dev-util/ragel
+"
+
+src_prepare() {
+	default
+
+	[[ ${PV} == 9999 ]] && eautoreconf
+
+	if use readline ; then
```

Comment:
In which case it should also be applied before eapply_user is being called, I suppose?